scripts/sommelier

```diff
@@ -67,6 +67,49 @@ init_wine() {
       mv "$WINEPREFIX" "${WINEPREFIX}.bak-$(date +%s)"
   fi
 
+  if [[ -n "${SOMMELIER_INSTALL_MONO:-}" ]]; then
+    if [[ $(grep -c mscoree "$WINEPREFIX"/user.reg 2&>/dev/null) -eq 0 ]]; then
+        WINE_VER=$(strings "$WINE_PLATFORM"/lib/libwine.so.1 | grep "wine-[\.0-9]" | cut -d "-" -f 2)
```

Comment:
Indentation is wrong for most of this block; should be two spaces less. Indentation is 2 spaces, not 4.